axelor-account/src/main/java/com/axelor/apps/account/web/InvoiceLineController.java

```diff
@@ -620,4 +622,50 @@ public void recomputeTax(ActionRequest request, ActionResponse response) {
       TraceBackService.trace(response, e);
     }
   }
+
+  public void onSelectTaxLineSet(ActionRequest request, ActionResponse response) {
+    try {
+      Context parentContext = request.getContext().getParent();
+
+      if (parentContext == null) {
+        return;
+      }
+
+      Invoice invoice = parentContext.asType(Invoice.class);
```

Comment:
why not call `getInvoice(Context)` which is doing the same thing but with more checks ?